website/src/doc-demos/geo-layers.js

```diff
@@ -17,22 +17,24 @@ import {DATA_URI} from '../constants/defaults';
 const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
```

Comment:
I think we can remove this line, since we're not using mapbox tiles at the MVTLayerDemo